uci.go

```diff
@@ -148,6 +148,16 @@ func (eng *Engine) SetFEN(fen string) error {
 	return err
 }
 
+// makeMove takes a move in UCI format (e.g e2e4 e5e7...) and tells the engine to set the position
+func (eng *Engine) makeMove(move string) error{
```

Comment:
this should probably be a public method.

Reply:
aac5d094f566312692ffcacfc4dbac4220d727cc I renamed it--thanks for the contribution!